pause_menu.py

```diff
@@ -0,0 +1,82 @@
+import curses
+
+
+def read_input(window):
+    return window.getch()
+
+
+class pause_menu:
+    def __init__(self, screen):
+        self.menu_items = ["[Z] Zurück zur Karte",
+                           "[S] Speicherstand laden", "[Q] Spiel verlassen"]
+        self.screen = screen
+        self.pressed_key = ord('z')
+
+    def print(self):
+        self.screen.clear()
+        size = self.screen.getmaxyx()
+
+        menu_item_win = curses.newwin(size[0], size[1], 0, 0)
+        yPosOffset = int(size[0] / 2) - 2
```

Comment:
Integer Division funktioniert in Python mit `//`, also `size[0] // 2` statt `int(...)`